File: largest_number_in_an_array.py
def largest_number_in_array(arr):
    max = arr[0]
    for i in range(1,len(arr)):
        if (arr[i]>max):
            max = arr[i]
    return max

arr = [int(i) for i in input("enter number into the array: ").split()]
result = largest_number_in_array(arr)
print("the largest number is: "+str(result))        

File: largest_number_in_an_array.js
function largest(arr) {
    let max = arr[0];
    for (let i = 1; i < arr.length; i++) {
        if (arr[i] > max) {
            max = arr[i];
        }
    }
    return max;
}

let arr = [10, 324, 45, 90, 9808];
console.log("Largest in given array is " + largest(arr));